shared/fs/banner/public-reminder.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+import openUrl from '../../util/open-url'
+
+type Props = {|
+  path: Types.Path,
+|}
+type State = {|
+  closed: boolean,
+|}
+
+const getURL = (path: Types.Path) => {
+  const parsedPath = Constants.parsePath(path)
+  switch (parsedPath.kind) {
+    case 'group-tlf':
+    case 'in-group-tlf':
+      return `https://keybase.pub/${parsedPath.tlfName}`
+    default:
+      return ''
+  }
+}
+
+class PublicBanner extends React.Component<Props, State> {
+  state = {closed: false}
+
+  _onClose = () => this.setState({closed: true})
+
```

Comment:
I think this works only because we never have scenario where we go directly from one public TLF to another -- in which case this component will get be kept mounted and maintain the closed state, thus fail to bring back the banner. Seems we could have a `componentDidUpdate` here to check if path change results in TLF change, just to be safe.

We could also add something like "lastDismissedPublicBannerTlf" in the redux store, depending on when exactly the banner is supposed to come back. For example, should it come back if user is in the Files tab, goes to chat, and comes back to the Files tab and land in the same public folder? Should it come back if user goes back out to the TLF list from a public folder, then back into the same public folder?

Reply:
Hmm, this is a good point. I'll put it in the redux store.

> For example, should it come back if user is in the Files tab, goes to chat, and comes back to the Files tab and land in the same public folder?   
I guess no here.

> Should it come back if user goes back out to the TLF list from a public folder, then back into the same public folder?
Not sure here, I'd be okay with yes. WDYT?

Reply:
Not sure! @cecileboucheron wdyt?

Reply:
hmm, so upon trying to implement this I am finding that the redux solution adds a lot of complication and doesn't actually seem to yield a consistent solution that I can think of. I think I'm going to go with the simpler state version with `componentDidUpdate` for now.

Reply:
I think it still depends on the behavior we want. If we want the "dismissed" state persist through screens, using the state wouldn't work as we'd lose states after unmount happens.